File: CSES Problem Set/Introductory Problems/Missing Number/src/code/java/missingNumber.java
import java.util.*;
public class missingNumber {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        long n = sc.nextLong();
        long sum = n*(n+1)/2;
        long temp =0;
        for (int i = 1; i < n; i++) {
            long k = sc.nextLong();
            temp = temp+k;
        } 
        
        long ans = sum - temp;
        System.out.println(ans);
        sc.close();
    }
    
}

File: CSES Problem Set/Introductory Problems/Missing Number/src/code/c++/missingNumber.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    long n;
    cin>>n;
    long sum = n*(n+1)/2;
    long temp =0;
    for (int i = 1; i < n; i++)
    {
        long k;
        cin>>k;
        temp = temp+k;
    }
    long ans = sum-temp;
    cout<<ans;
    
}